### PR TITLE
Added app description to the form pages and added more context to the subscription status

### LINKED ISSUE #23: Add some explanatory text to login and signup on what the service does
On both the login and signup screens, add a short line to explain to people what they are signing up for

e.g.
"Get Nasa's Astronomy Picture of the day texted to your phone every day"

### LINKED ISSUE #24: Add more context to subsctription status
The copy "You are not subscribed to nasa APOD" should be more verbose.

e.g.
You are currently subscribed. You will get a text with Nasa's Astronomy Picture of the Day every day at {time}

You are currently unsubscribed. Subscribe below to get a text with Nasa's Astronomy Picture of the Day every day at {time}

### PR DESCRIPTION
**Pull request by Daniel-olaO**
close #23 
close #24

### CODE REVIEW

File: src/components/Dashboard.jsx [hunk 1]
```diff
@@ -70,10 +70,15 @@ const Dashboard = ({setIsAuthenticated}) => {
       <Container maxWidth="sm">
         {loading ? <Alert severity="info">Loading...</Alert> :
           isSubscribed ? <Alert severity="info">
-            You are subscribed to nasa APOD
+            You are currently subscribed.
+            You will get a text with Nasa's Astronomy
+            Picture of the Day every day at 12:00 PM.
```
Comment (Russell-Pollari) [inline src/components/Dashboard.jsx]: what time zone? 

servers usually run on UTC time. 

Quick fix: specify time's zone
Better fix: display in the client's timezone

File: src/components/Description.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {Alert} from '@mui/material';
+
+const Description = () => {
+  return (
+    <Alert severity="info" className='app-desc'>
+      This app get NASA's Astronomy Picture of the Day (APOD)
+       and text it to users' phones everyday.
```
Comment (Russell-Pollari) [inline src/components/Description.jsx]: watch for typos.. this app get**s**
text**s** it

Also, you should write for the user, not about them. Pretend we want people using this

"Get NASA's Astronomy Picture of the Day (APOD) in a daily text"

**Review by Russell-Pollari — CHANGES_REQUESTED** [review]

File: src/components/Description.jsx [hunk 3]
```diff
@@ -4,8 +4,8 @@ import {Alert} from '@mui/material';
 const Description = () => {
   return (
     <Alert severity="info" className='app-desc'>
-      This app get NASA's Astronomy Picture of the Day (APOD)
-       and text it to users' phones everyday.
+      "Get NASA's Astronomy Picture of the Day (APOD)
```
Comment (Russell-Pollari) [inline src/components/Description.jsx]: Why the quotes? I'd say remove

Also typo => in **a** daily text

**Review by Russell-Pollari — CHANGES_REQUESTED** [review]

File: src/components/Description.jsx [hunk 4]
```diff
@@ -4,8 +4,8 @@ import {Alert} from '@mui/material';
 const Description = () => {
   return (
     <Alert severity="info" className='app-desc'>
-      "Get NASA's Astronomy Picture of the Day (APOD)
-       in daily text"
+      "Get NASA Astronomy Picture of the Day (APOD)
```
Comment (Russell-Pollari) [inline src/components/Description.jsx:7]: Remove the quotes. Unless you want to keep them for some reason?

**Review by Russell-Pollari — CHANGES_REQUESTED** [review]
remove quotes

File: src/components/Description.jsx (hunk 4 above)
Comment (Daniel-olaO) [reply src/components/Description.jsx:7]: I don't mind

Comment (Russell-Pollari) [reply src/components/Description.jsx:7]: Ok.. fair enough.

**Review by Russell-Pollari — APPROVED** [review]

**Pull request merged**